management-interface.go

```diff
@@ -215,6 +218,33 @@ func CheckInterfaceHandler(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(response)
 }
 
+// SpeakerTestHandler will show a frame from the camera to help with positioning
+func SpeakerTestHandler(w http.ResponseWriter, r *http.Request) {
+	tmpl.ExecuteTemplate(w, "speaker-test.html", nil)
+}
+
+// SpeakerStatusHandler attempts to play a sound on connected speaker(s).
+func SpeakerStatusHandler(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
+	response := make(map[string]string)
+
+	// fileName := soundsBox.String("IEEE_float_mono_32kHz.wav")
+	fileName := "./sounds/IEEE_float_mono_32kHz.wav"
```

Comment:
Referencing a path relative to the current directory is quite fragile. There are no guarantees about what the the working directory was when a program was started.

I suggest you have this look for the wav file in the same directory as the binary (see https://stackoverflow.com/a/18537419 for how to obtain this) and if it's not there look in `/usr/ilb/management-interface`/). This then makes it convenient for both development and release scenarios.